__tests__/api.test.js

```diff
@@ -230,6 +230,81 @@ describe('POST /api/articles/:article_id/comments', () => {
     })
   });
 });
+
+describe('PATCH /api/articles/:article_id', () => {
+  test('200: updates an article by the given id and responds with the updated article', () => {
+    const updateBody = {
+      inc_vote: 100
+    }
+    return request(app)
+    .patch('/api/articles/1')
+    .send(updateBody)
+    .expect(200)
+    .then(({ body })=> {
+      const { updatedArticle } = body;
+      expect(updatedArticle).toMatchObject({
+          article_id: 1,
+          title: 'Living in the shadow of a great man',
+          topic: 'mitch',
+          author: 'butter_bridge',
+          body: 'I find this existence challenging',
+          created_at: '2020-07-09T20:11:00.000Z',
+          votes: 200,
+          article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+      })
+    })
+  });
+
+  test('400: responds with "Bad request" when Body is missing fields', () => {
```

Comment:
Consider if you might want to make other patch functionality in the future (e.g. patching title), in that case this test would need to be slightly altered. However, it works well for the time being :)

Reply:
Thank you for the feedback. I will consider the other patch functionality and adjust accordingly. :)